kahuna/app/views/main.scala.html

```diff
@@ -2,7 +2,8 @@
   authApiUri: String,
   reauthUri: String,
   sentryDsn: Option[String],
-  sessionId: String)
+  sessionId: String,
+  googleTrackingId: String)
```

Comment:
Can we make this an `Option[String]`? That way we can avoid the empty string comparison below.